mms/metrics/dimension.py

```diff
@@ -0,0 +1,43 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+"""
+Dimension class for model server metrics
+"""
+
+from collections import OrderedDict
```

Comment:
this is just to keep reading of the metrics uniform, say in a log, different keys are ordered differently to the program it doesn't matter, but for those looking though logs this will help.